virtcontainers/qemu.go

```diff
@@ -573,6 +575,34 @@ func (q *qemu) startSandbox(timeout int) error {
 		}
 	}()
 
+	if q.config.VirtioFS {
+                var cmd *exec.Cmd
+		sockPath, err := utils.BuildSocketPath(runStoragePath, q.id, "vhost-fs.sock")
+		if err != nil {
+			return err
+		}
+
+		sourcePath := filepath.Join(kataHostSharedDir, q.id)
+
+		var args []string
+		args = append(args, "-o", "virtio_socket=" + sockPath,
+				    "-o", "source=" + sourcePath,
+				    "-o", "cache=" + q.config.VirtioFSCache)
+		if q.config.VirtioFSSharedVersions {
+			args = append(args, "-o", "shared")
+		}
+		args = append(args, "/")
```

Comment:
Q. why `/` ? are you given access to the whole rootfs? can you please add a comment?

Reply:
This is removed in the next revision of this PR.  The "/" was a dummy argument that has now been removed from virtiofsd.  The shared directory is specified by the `virtiofsd -o source path/to/dir` option.